Documentation/docs-mobile/building-apps/build-items.md

```diff
@@ -129,6 +129,35 @@ multiple files, and they will be evaluated in no particular order (so don't
 specify the same environment variable or system property in multiple
 files).
 
+## AndroidGradleProject
+
+`<AndroidGradleProject>` can be used to build and consume the outputs
+of Android Gradle projects created in Android Studio or elsewehere.
+
+The `Include` metadata should point to the root `build.gradle` file that will
```

Comment:
"…to the root `build.gradle` or `build.gradle.kts` file…"

At least, I assume we can use `build.gradle.kts` scripts?

Reply:
Updated this to improve clarity.